webapp/src/main.tsx

```diff
@@ -2,13 +2,18 @@
 // See LICENSE.txt for license information.
 import React from 'react'
 import ReactDOM from 'react-dom'
+import {store} from 'emoji-mart'
 
 import App from './app'
 import {initThemes} from './theme'
+import {importNativeAppSettings} from './nativeApp'
+import {UserSettings} from './userSettings'
 
 import './styles/variables.scss'
 import './styles/main.scss'
 import './styles/labels.scss'
 
+store.setHandlers({getter: UserSettings.getEmojiMartSetting, setter: UserSettings.setEmojiMartSetting})
+importNativeAppSettings()
```

Comment:
I moved this here because the `initThemes` call below already reads from localStorage.